internal/controller/common/test_common.go

```diff
@@ -72,8 +73,10 @@ var (
 )
 
 func CreatePipelineRolloutInK8S(ctx context.Context, t *testing.T, numaplaneClient client.Client, pipelineRollout *apiv1.PipelineRollout) {
+	pipelineRolloutCopy := *pipelineRollout
 	err := numaplaneClient.Create(ctx, pipelineRollout)
```

Comment:
the act of calling `Create()` clears the Status out, so we need to save a copy of the pipelineRollout so we have the Status